packages/nextjs/app/builders/0xB9784f1A5D59779a4df0D36453284f8457E53bDE/page.tsx

```diff
@@ -0,0 +1,97 @@
+import Image from "next/image";
+import airo from "./assets/uncle-airo.jpg";
+import { NextPage } from "next";
+import {
+  ChatBubbleLeftRightIcon,
+  CodeBracketIcon,
+  GlobeAltIcon,
+  LightBulbIcon,
+  RocketLaunchIcon,
+  ServerIcon,
+} from "@heroicons/react/24/outline";
+import { BuidlGuidlLogo } from "~~/components/assets/BuidlGuidlLogo";
+import { Address } from "~~/components/scaffold-eth";
+
+const Page: NextPage = () => {
+  return (
+    <>
```

Comment:
Nit: Since you already a wrapping parent `div` you could have omitted this empty fragment